config/application.rb

```diff
@@ -23,6 +23,7 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.autoload_paths << Rails.root.join("app", "uploaders")
+    config.autoload_paths << Rails.root.join("app", "decorators")
```

Comment:
Can you try accessing the decorator in the Rails console? I usually have problems with that, not with Rails server.